src/solver/mlir_common.cpp

```diff
@@ -60,6 +60,17 @@ const char* DTypeName(miopenDataType_t ty)
     MIOPEN_THROW(miopenStatusInternalError, "Value outside of datatype enum");
 }
 
+static std::string GetIsaName(const miopen::TargetProperties& target)
+{
+#if ROCM_FEATURE_TARGETID_OFF
+    const char* const ecc_suffix = (target.Sramecc() && *target.Sramecc()) ? ":sramecc+" : "";
+    return {"amdgcn-amd-amdhsa:" + target.Name() + ecc_suffix};
+#else
+    const LcOptionTargetStrings lots(target);
```

Comment:
Please add `#include <miopen/hip_build_utils.hpp> // for LcOptionTargetStrings`

Reply:
Done in e8de8d3